hack/install_kustomize.sh

```diff
@@ -123,31 +133,38 @@ s390x)
     ;;
 esac
 
-releases=$(curl -s $release_url)
+releases=$(curl -s "$release_url")
 
 if [[ $releases == *"API rate limit exceeded"* ]]; then
   echo "Github rate-limiter failed the request. Either authenticate or wait a couple of minutes."
   exit 1
 fi
 
-RELEASE_URL=$(echo "${releases}" |\
-  grep browser_download.*${opsys}_${arch} |\
-  cut -d '"' -f 4 |\
-  sort -V | tail -n 1)
+RELEASE_URL="$(find_release_url "$opsys" "$arch")"
+
+if [[ "$(uname -m)" == "aarch64" ]]; then
+    # fallback to the old behavior of downloading amd64 binaries on aarch64 systems.
+    # People might have qemu-binfmt-misc installed, so it worked for them previously.
+    echo "Version $version does not exist for ${opsys}_arm64, trying ${opsys}_amd64 instead."
+    RELEASE_URL="$(find_release_url "$opsys" "amd64")"
```

Comment:
This is currently always replacing the arm64 version. I think you wanted something like:

```bash
if [[ "$(uname -m)" == "aarch64" ]] && [[ -z "$RELEASE_URL" ]]; then
    # fallback to the old behavior of downloading amd64 binaries on aarch64 systems.
    # People might have qemu-binfmt-misc installed, so it worked for them previously.
    echo "Version $version does not exist for ${opsys}_arm64, trying ${opsys}_amd64 instead."
    arch=amd64
    RELEASE_URL="$(find_release_url "$opsys" "$arch")"
fi
```

Reply:
Yes, indeed. Updated.